lib/statix/conn.ex

```diff
@@ -42,6 +42,12 @@ defmodule Statix.Conn do
         if(is_atom(sock), do: "", else: "Statix ") <>
           "#{inspect(sock)} #{type} metric \"#{key}\" lost value #{val}" <> " due to port closure"
       end)
+
+      # open a new port for subsequent tranmissions
+      # NOTE: this port will be linked to the process that
+      # called `transmit`, so perhaps this should be handled
+      # by sending a message to a GenServer instead?
+      open(conn)
```

Comment:
If necessary, we could use `:gen_udp.controlling_process/2` to reassign the parent process.